packages/atomic/src/pages/test.html

```diff
@@ -18,11 +18,13 @@
           accessToken: 'xx564559b1-0045-48e1-953c-3addd1ee4457',
           organizationId: 'searchuisamples',
         });
+        searchInterface.executeFirstSearch();
       })();
     </script>
   </head>
+
   <body>
-    <atomic-search-interface id="search" reflect-state-in-url="false">
+    <atomic-search-interface id="search" reflect-state-in-url="true" sample>
```

Comment:
We can just remove the `reflect-state-in-url` if its set to true since that's the default.

Reply:
Also @ThibodeauJF do we need both the JS instantiation & the sample tag? I kept it the same as inside `index.html` which only has the JS.